conjure-java-undertow-runtime/src/main/java/com/palantir/conjure/java/undertow/runtime/ConjureHandler.java

```diff
@@ -141,8 +166,20 @@ public Builder addWrapperBeforeBlocking(EndpointHandlerWrapper wrapper) {
             return this;
         }
 
+        /** Configures the runtime instance to use with this handler. */
+        @CanIgnoreReturnValue
+        public Builder runtime(UndertowRuntime value) {
+            this.runtime = Preconditions.checkNotNull(value, "UndertowRuntime");
+            return this;
+        }
+
         public HttpHandler build() {
-            checkOverlappingPaths();
+            ImmutableList<Endpoint> allEndpoints = Streams.concat(
+                            endpoints.stream(), services.stream().flatMap(service -> service.endpoints(runtime).stream()
+                                    .map(Builder::checkEndpoint)))
+                    .collect(ImmutableList.toImmutableList());
```

Comment:
nit: find it clearer to have a separate variable for endpoints from a service
```suggestion
            Stream<Endpoint> serviceEndpoints = services.stream()
                    .flatMap(service -> service.endpoints(runtime).stream().map(Builder::checkEndpoint));
            ImmutableList<Endpoint> allEndpoints =
                    Streams.concat(endpoints.stream(), serviceEndpoints).collect(ImmutableList.toImmutableList());
```